traffic_portal/test/integration/specs/CacheGroup.spec.ts

```diff
@@ -41,20 +41,38 @@ cachegroups.tests.forEach(cacheGroupData => {
                 await cacheGroupPage.OpenTopologyMenu();
                 await cacheGroupPage.OpenCacheGroupsPage();
             })
+            cacheGroupData.check.forEach(check => {
+                it(check.description, async () => {
+                    expect(await cacheGroupPage.CheckCSV(check.Name)).toBeTruthy();
```

Comment:
`CheckCSV` returns a `Promise<boolean>`, so this can check specifically for `.toBeTrue()`, not merely `.toBeTruthy()`. Or you could use `.toBe(true)`, the difference between that and `.toBeTrue()` is convoluted and involves using `Boolean` as an ES6 constructor - which you should never do anyway.

Reply:
Fixed.